modules/backend/formwidgets/permissioneditor/assets/css/permissioneditor.css

```diff
@@ -4,7 +4,7 @@
 .permissioneditor table{width:100%}
 .permissioneditor table th{padding:30px 4px 8px 4px;color:#2a3e51;font-weight:normal;border-bottom:1px solid #dbe1e3}
 .permissioneditor table th.tab{font-size:13px}
-.permissioneditor table th.permission-type{text-transform:uppercase;font-size:11px;text-align:center;cursor:pointer}
+.permissioneditor table th.permission-type{text-transform:uppercase;font-size:11px;text-align:center}
```

Comment:
Why was this removed?

Reply:
It's not in the source LESS file. I probably ran the old util compile at some point.

Reply:
Should we add it back in?

Reply:
Evidentally, it was never in there. @marvindurot added that CSS in https://github.com/wintercms/winter/pull/936 but we forgot to get it included in the LESS file. I'll re-add and re-gen.